redcap/methods/surveys.py

```diff
@@ -11,6 +11,58 @@
 class Surveys(Base):
     """Responsible for all API methods under 'Surveys' in the API Playground"""
 
+    def export_survey_link(
+        self,
+        instrument: str,
+        record: str,
+        event: Optional[str] = None,
+        repeat_instance: int = 1,
+    ):
+        """
+        Export one survey link
+
+        Note:
+            The passed instrument must be set up as a survey instrument.
+
+        Args:
+            instrument:
+                Name of instrument as seen in the Data Dictionary (metadata).
+            record:
+                Name of the record
+            event:
+                Unique event name, only used in longitudinal projects
+            repeat_instance:
+                only for projects with repeating instruments/events) 
+                The repeat instance number of the repeating event (if longitudinal) 
+                or the repeating instrument (if classic or longitudinal). 
+                Default value is '1'.
+
+        Returns:
+            Str:
+                URL of survey Link requestet
+
+        Examples:
+            >>> proj.export_survey_link(instrument="form_1", record="5", event="event_1_arm_1")
+            https://redcap.mytld.com/surveys/?s=6B2zFAEWPVSrXnnx
+        """       
+        payload = self._initialize_payload(
+            content="surveyLink"
+        )
+        payload["instrument"] = instrument
+        payload["record"] = record
+        if event:
+            payload["event"] = event
+        payload["repeat_instance"] = repeat_instance
+
+        return_type = "str"
+        response = cast(Union[Json, str], self._call_api(payload, return_type))
```

Comment:
This totally works, the `_lookup_return_type` helper is more meant for functions that could return different types depending on if the call has a `format` parameter in the API call. Since this one always returns a string, the implementation here is great.